docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,181 @@
+In order to provide more artistic control, some key parameters of the AdnSkin solver are exposed as paintable attributes in the deformer. The [AdonisFX Paint Tool](tools.md#adonisfx-paint-tool) must be used to paint those parameters to ensure that the values satisfy the solver requirements.
 
-- *Hard Constraints*: weight to modulate the correction applied to the vertices to keep them at a constant transformation local to the closest point on the reference mesh at initialization. The recommendation for a biped or quadruped creature is to use a maximum value of 1.0 on the wrists, ankles and hips and a value of 0.2 on the rest of the body.
-- *Slide Constraints*: weight to modulate the correction applied to the vertices to keep them at a constant distance to the reference mesh sliding along the reference surface. In the example of a biped or quadruped, we recommend a value of 1.0 on the scapulas, shoulders, elbows and knees and a value of 0.2 on the rest of the body.
-- *Soft Constraints*: weight to modulate the correction applied to the vertices to keep them at a constant distance to the closest point on the reference mesh at initialization. An intermediate value of 0.5 on the whole geometry is recommended.
-- *Compression Resistance*: force to correct the edge lengths if the current length is smaller than the rest length. A higher value represents higher correction.
-- *Stretching Resistance*: force to correct the edge lengths if the current length is greater than the rest length. A higher value represents higher correction.
+- **Hard Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant transformation, local to the closest point on the reference mesh at initialization. Hard Constraint maps will force the geometry points to keep the original position. A low value of *Hard Constraints* may be desired to allow the skin to create wrinkles and sliding effect.
 
-<figure>
-  <img src="images/skin_paint_example.png" caption="AdonisFX Paint Tool"> 
-  <figcaption>Figure 1: Example of painted weights on the skin of a bear character. From left to right: Hard Constraints, Slide Constraints and Soft Constraints.</figcaption>
-</figure>
+    - *Tip*: Flood the geometry with a very low value 0.1 - 0.2. Give a value of 1.0 to the edges of the skin to guarantee that is properly attached to the target geometry.
 
-> [!NOTE]
-> - *Hard*, *Soft* and *Slide* values are normalized for each vertex. Make sure to paint the values that you want to give priority to at the end in order to avoid the internal normalization override them in further strokes.
+    - *Tip*: Smooth the borders by using the Smooth and Flood combination to make sure the simulation does not find hard edges as this could create odd wrinkles (unless that is something you are looking for!).
```

Comment:
We agreed to remove expressions like "odd". Something like this: (please, review the whole document)
```suggestion
    - *Tip*: Smooth the borders by using the Smooth and Flood combination to make sure that there are no discontinuities in the weights map. This will help the simulation to not produce sharp differences in the dynamics of every vertex compared to its neighbors vertices (unless that is something you are looking for!).
```

Reply:
Applied this change! Also reviewed the whole document to avoid subjective expressions.

Reply:
I think we should say: "neighboring vertices" or "connected vertices".